### DISCUSSION TITLE
Installation Troubles: File Not Found error for CMAKE and msise00_driver.exe

### DISCUSSION THREAD

**Discussion by aprater11** [scienceopen/msise-00#27] [Q&A]
Hello scivision, 
I'm a python amateur and am having some troubles with the msise00 installation and initialization. After following all of the steps in the readme, and installing MSYS2 64bit, I'm hitting the following errors when I try to call msise00 functions: 
`FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Python37\\lib\\site-packages\\msise00\\msise00_driver.exe'`
and
`FileNotFoundError: CMake not available`

As far as I can tell, `msise00_driver.exe` is not included in the base package, so it's searching for a non existent file. I've verified that this .exe is not in my site-packages (or anywhere on my C: drive). Is this something that gets generated by the package? 
 
Running `pytest msise00` produces similar failures reports regarding inability to locate `msise00_driver.exe` and that `CMAKE is not available`.

I'd appreciate any support you're able to provide on this issue. Thanks! 
austinprater11@gmail.com

Full err output below:
```
Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\msise00\base.py", line 165, in rungtd1d
    with importlib.resources.path(__package__, exe_name) as exe:
  File "C:\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Python37\lib\importlib\resources.py", line 201, in path
    with open_binary(package, resource) as fp:
  File "C:\Python37\lib\importlib\resources.py", line 91, in open_binary
    return reader.open_resource(resource)
  File "<frozen importlib._bootstrap_external>", line 929, in open_resource
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Python37\\lib\\site-packages\\msise00\\msise00_driver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:/Users/Austin Prater/Desktop/araft/test.py", line 18, in <module>
    atmos = msise00.run(time=datetime(2013, 3, 31, 12, 0), altkm=z, glat=lat, glon=lon)
  File "C:\Python37\lib\site-packages\msise00\base.py", line 64, in run
    atmos = rungtd1d(time, altkm, glat.squeeze()[()], glon.squeeze()[()], indices)
  File "C:\Python37\lib\site-packages\msise00\base.py", line 168, in rungtd1d
    build()
  File "C:\Python37\lib\site-packages\msise00\base.py", line 36, in build
    raise FileNotFoundError("CMake not available")
FileNotFoundError: CMake not available
```

**Comment by danielk333** [comment]
It seems the problem here is that you do not have [CMake](https://cmake.org/) installed. It is needed in order to build the driver `msise00_driver.exe`.

To manually attempt building you can start a prompt and run
```python
>>> import msise00.base
>>> msise00.base.build()
```

**Comment by scivision** [comment]
Correct @danielk333 thank you
